apps/hash-graph/lib/graph/src/api/rest.rs

```diff
@@ -144,6 +144,20 @@ pub fn rest_api_router<P: StorePool + Send + 'static>(
         )
 }
 
+pub fn openapi_only_router() -> Router {
+    let open_api_doc = OpenApiDocumentation::openapi();
+
+    Router::new()
+        .route(
+            "/api-doc/openapi.json",
+            get({
+                let doc = open_api_doc;
+                move || async { Json(doc) }
+            }),
+        )
+        .route("/api-doc/models/*path", get(serve_static_schema))
```

Comment:
```suggestion
    Router::new().nest(
        "/api-doc",
        Router::new()
            .route(
                "/openapi.json",
                get({
                    let doc = open_api_doc;
                    move || async { Json(doc) }
                }),
            )
            .route("/models/*path", get(serve_static_schema)),
    )
```

Reply:
[Reuse openapi_only_router](https://github.com/hashintel/hash/pull/2358/commits/b0f04c1c947054dde6e7c563204545692109278d)

Reply:
Oops didn't notice this was actually a change to the nesting

Reply:
[Unify nesting of OpenAPI spec and fix bug](https://github.com/hashintel/hash/pull/2358/commits/7b713ae80319496cc1626f576a5006a48d5179d3)